pybio/torch/transformations/augmentations.py

```diff
@@ -14,5 +14,5 @@ def set_next_state(self):
         self.k = np.random.randint(0, 4)
 
     # FIXME only works for 2d inputs
-    def apply_transformation(self, tensor):
+    def apply_to_one(self, tensor):
```

Comment:
Technically this does not depend on pytorch either, so could go to `pybio`.

Reply:
yes, should we move SynchronizedTransformation to `pybio.core` as well then?
And what about assuming the Transformation is independent (remove additional IndpendentTransformation)?